index.html

```diff
@@ -3950,42 +3950,197 @@ <h2>Syntaxes</h2>
+of an in-message association with its terminology definitions. Details on
+processing VCDM documents consistently with the context file, but without
+[[!JSON-LD]] tooling, can be found in the "JSON Processing" section below.
+          </p>
+
+          <p>
+The referenced context file provides the foundation for a consistent
+grammatical layer based on object-oriented modeling and community driven
+terminology selection which enables coexistence of various JSON tree shapes and
+colloquial terms while preventing likely terminology collisions or creating a
+need to write new tree traversal code for every new media type or JSON shape or
+parsing approach.
+          </p>
+
+          <p>
+The Verifiable Credentials Data Model uses this consistent foundation to provide a
+common layer across all credentials, presentations, and transaction formats
+within a single, consistently parseable media type definition. This allows for
+ecosystem extensibility without requiring a centralized registry to be updated
+for every new application or community. The core data model terms are
+registered and specified while communities are free to build on that foundation
+through their own specification process. The terminology mappings are
+referenced from the VCDM document removing the need for an ever changing
+landscape of "private claims" or narrowly defined "public claims" housed in a
+centralized registry. The common grammar and object-oriented data modeling
+remain the same and properly contextualized terms can commingle. Applications
+and communities can extend what is claimed and signed through the use of
+vocabularies.
+          </p>
+
+          <p>
+The underpinning graph data model enables signing of the document's statements
+rather than just the envelope in which it's contained in. When using this feature, the
+data — while signed — is accessible in the same way it had been prior to signing,
+allowing for easier indexing, storage, and in-Wallet selection. The existence of
+the proof within the object-oriented graph model provides confidence that not
+only the wrapping structure was signed, but its actual informational contents.
+This enables selective disclosure of statements within the same JSON format
+avoiding the ever increasing cost of retooling for new disclosure formats or
+new envelope types. The proof is consistently placed alongside the claims.
+          </p>
+
+          <p>
+[[!JSON-LD]] provides Verifiable Credentials with a consistent grammatical
+foundation, object-oriented modeling, and collision-free terminology sharing
+which allows for ecosystem growth, document format iteration (rather than
+reinvention), and an extensible framework for new cryptographic proof
+mechanisms without altering or damaging the underlying data structure. This
+creates economies of scale which reduces development time, decreases
+retooling cost, decreases time to market, and provides long term stability for
+Issuers, Verifiers, and Wallets.
```

Comment:
```suggestion
mechanisms without altering or damaging the underlying data structure.
```

Reply:
Shared foundational data models and consistent data processing pipelines do quantifiably reduce development time, decrease retooling cost, etc. Could you expand on why you'd like this statement removed?

Reply:
@Sakurann this paragraph remains unchanged in the most recent revision, so please let me know if there are specific complaints with the above claims. Thanks!